contributor-guide/modules/ROOT/pages/version-control.adoc

```diff
@@ -1,4 +1,57 @@
-= Version Control (Git)
+= Version Control
 :navtitle: Version Control
 
-*Content under construction.*
+Boost uses the Git version control system for its development. This allows multiple contributors to work on the libraries concurrently, while keeping track of the changes that each person makes. Git is a distributed version control system, which means each contributor has their own local copy of the entire project, including its history.
+
+== Library Repositories
+
+The Boost libraries are modular, and each library has its own separate repository.  This makes it easier to work with the entire collection of Boost libraries.
+
+The official Boost repositories are hosted on GitHub at https://github.com/boostorg/[Boost.org]. Each library has its own repository under this organization. For example, https://github.com/boostorg/json for boost:json[], or https://github.com/boostorg/date_time/ for boost:date_time[].
+
+As with any Git-based project, you can clone the repositories, make changes, commit those changes to your local repository, and push your changes back to the server. If you want to contribute changes to the official Boost libraries, you can make a pull request, which the maintainers of the libraries can then review and potentially merge into the official codebase. Be aware though that Boost has a well-defined workflow for contributors, including guidelines for how to submit changes, how changes are reviewed, and so forth. Refer to xref:getting-involved.adoc[].
+
+== The Super-project Repository
+
+The Boost https://github.com/boostorg/boost[*Super-project*] repository includes all the libraries as submodules.
+
+In addition to the libraries, you'll find many meta-repositories under `boostorg` that handle other tasks, for example:
+
+* https://github.com/boostorg/website2022[Website generation]
+* https://github.com/boostorg/cmake[Cmake support]
+* https://github.com/boostorg/regression[Regression testing]
+* https://github.com/boostorg/release-tools[Release tools]
+
+There is also a https://github.com/boostorg/boost/tree/master/tools[tools] directory in the Super-project, which contains various tools used for tasks such as building and testing Boost libraries. 
+
+The *master* branch of the Super-project is usually the development version of Boost, where new features are added and bugs are fixed. When there is a public release of Boost, it is built from the Super-project repo.
+
+== Bug and Issue Reporting
```

Comment:
Isn't "Bug and Issue Reporting" more related to the user guide?

Reply:
Yeah, this extra info should go under "Reporting Issues" in User guide.

I think there's room for a section like this here, but geared towards library authors. Probably speaking on how to create an issue template in GitHub to guide users and make the process smoother. If you prefer, I can take the lead on this, apply it to MySQL and submit a PR with whatever I find better.